paasta_tools/kubernetes/application/controller_wrappers.py

```diff
@@ -50,7 +50,8 @@ def __init__(
 
         replicas = (
             item.spec.replicas
-            if item.spec.template.metadata.annotations.get("autoscaling") is None
+            if item.metadata.labels.get(paasta_prefixed("autoscaled"), "false")
```

Comment:
How come it's at a different level to the annotations? Is that because it's the Deployments labels rather than the Pod lables?

Reply:
i'm actually not sure - i was going off off the other label accesses above...lemme run this in kubestage again real fast and see if labels are also available in the template spec or if its just annotations in there for whatever reason

Reply:
huh, looks like i missed something when refactoring and didn't actually also set this label on the deployment (and just did the pods) - i'll add a follow-up commit that correctly refactors this

Reply:
done - good catch! we should probably eventually add an itest/e2e test for s_k_j since my existing unit test was mocking things out in the expected format - but the actual code i pushed didn't match that format when run irl :p